crates/kubelet/src/config.rs

```diff
@@ -178,6 +193,13 @@ pub struct Opts {
     )]
     node_ip: Option<IpAddr>,
 
+    #[structopt(
+        long = "node-labels",
+        env = "KRUSTLET_NODE_LABELS",
```

Comment:
I agree it's more concise without the `KRUSTLET_` prefix and this will be more consistent how `kubelet` works. I renamed this to `NODE_LABELS`. I will let the rest of the `KRUSTLET_` prefixed flags untouched for now.